tests/test_gabbits.py

```diff
@@ -36,6 +36,8 @@ def __init__(self):
         self.environ = {
             "ROCKET_ADDRESS": XSNIPPET_API_HOST,
             "ROCKET_PORT": str(XSNIPPET_API_PORT),
+            "ROCKET_LOG": "critical",
```

Comment:
Does Rocket itself uses `tracing`?

Reply:
Not yet. It should in release 0.5, but it's not clear when that happens, and the corresponding PR hasn't been merged yet.